src/stream/src/common/log_store_impl/kv_log_store/buffer.rs

```diff
@@ -28,7 +28,7 @@ use crate::common::log_store_impl::kv_log_store::{
 };
 
 #[derive(Clone)]
-pub(crate) enum LogStoreBufferItem {
+pub enum LogStoreBufferItem {
```

Comment:
The `synced_kv_log_store` is also in the same crate, so not need to extend the visibility.